rust/blockstore/src/arrow/block/delta/delta.rs

```diff
@@ -58,44 +57,45 @@ impl BlockDelta {
     ///  the arrow specification. When a block delta is converted into a block data
     ///  the same sizing is used to allocate the memory for the block data.
     pub(in crate::arrow) fn get_size<K: ArrowWriteableKey, V: ArrowWriteableValue>(&self) -> usize {
-        let prefix_data_size = self.builder.get_prefix_size(0, self.len());
-        let key_data_size = self.builder.get_key_size(0, self.len());
-        let value_data_size = self.builder.get_value_size(0, self.len());
-
-        self.get_block_size::<K, V>(
-            self.builder.len(),
-            prefix_data_size,
-            key_data_size,
-            value_data_size,
-        )
+        // let prefix_data_size = self.builder.get_prefix_size();
```

Comment:
is this left on purpose or safe to delete?

Reply:
my bad, got missed in cleanup, thanks

Reply:
https://github.com/chroma-core/chroma/pull/2684/commits/925013bbd68df536027ab9783f829d41636af642

fixed